package/MDAnalysis/coordinates/PDB.py

```diff
@@ -465,6 +476,8 @@ class PDBWriter(base.WriterBase):
     The maximum frame number that can be stored in a PDB file is 9999 and it
     will wrap around (see :meth:`MODEL` for further details).
 
+    If unit cell dimensions are not set, unitary values are used and an
```

Comment:
Add explicit explanation what unitary values means: cubic box with sides 1 Å.

Link to the PDB specs that we are following. Say that we are following PDB standard here. It will make it sooo much easier when we start answering questions...

Reply:
This is the place where docs are really needed, basically the ones on CRYST1 that you had above.